Assets/Script/Venue/BundleBackgroundManager.cs

```diff
@@ -20,6 +20,8 @@ public class BundleBackgroundManager : MonoBehaviour
         public const string BACKGROUND_SHADER_BUNDLE_NAME = "_metal_shaders.bytes";
         public const string BACKGOUND_OSX_MATERIAL_PREFIX = "_metal_";
 
+        private const int VENUE_LAYER_NUMBER = 9;
```

Comment:
Given that Unity doesn't guarantee (afaik) that the layer number will never get silently changed, I think it would be better define the _name_ of the layer mask here and initialize the _number_ in `Awake()` with a `LayerMask.NameToLayer()` call.

If I'm wrong about that, lmk and I'll merge as-is.


```suggestion
        private       int    VENUE_LAYER_NUMBER = -1;
        private const string VENUE_LAYER_NAME = "Venue";
```